test/verify/check-storage-resize

```diff
@@ -133,6 +134,164 @@ class TestStorage(StorageCase):
                              can_shrink=False,
                              can_grow=False)
 
+    def testGrowShrinkHelp(self):
+        m = self.machine
+        b = self.browser
+
+        if self.storaged_version < [2, 7, 6]:
+            # No Filesystem.Size property
+            raise unittest.SkipTest("UDisks2 too old")
+
+        self.login_and_go("/storage")
+        m.add_disk("500M", serial="DISK1")
+        b.wait_in_text("#drives", "DISK1")
+
+        m.execute("vgcreate TEST /dev/disk/by-id/scsi-0QEMU_QEMU_HARDDISK_DISK1")
+        m.execute("lvcreate TEST -n vol -L 200m")
+        b.wait_present("#vgroups")
+        b.wait_in_text("#vgroups", "TEST")
+        b.click('#vgroups tr:contains("TEST")')
+        b.wait_visible("#storage-detail")
+        self.content_row_wait_in_col(1, 2, "/dev/TEST/vol")
+
+        self.content_tab_action(1, 2, "Format")
+        self.dialog_wait_open()
+        self.dialog_set_val("name", "FSYS")
+        self.dialog_set_val("type", "ext4")
+        self.dialog_apply()
+        self.dialog_wait_close()
+        self.content_tab_wait_in_info(1, 2, "Name", "FSYS")
+
+        mountpoint = self.mount_root + "/admin/FSYS"
+        self.content_tab_action(1, 2, "Mount")
+        self.content_tab_wait_in_info(1, 2, "Mounted At", mountpoint)
+
+        # Grow the logical volume and let Cockpit grow the filesystem
+
+        m.execute("lvresize TEST/vol -L+100M")
+        warning_tab = self.content_tab_expand(1, 3)
+
+        def confirm():
+            self.dialog_wait_open()
+            self.dialog_apply()
+            self.dialog_wait_close()
+            # HACK - https://github.com/storaged-project/udisks/pull/631
+            m.execute("udevadm trigger")
+
+        self.content_tab_action(1, 3, "Grow Content")
+        confirm()
+        b.wait_not_present(warning_tab)
+        size = int(m.execute("df -k --output=size %s | tail -1" % mountpoint).strip())
+        assert (size > 250000)
```

Comment:
Let's please use `self.assertGreaterThan(size, 250000)` here, so that we get more useful test failures.

Reply:
Done.